src/lucky_cli/task.cr

```diff
@@ -13,6 +13,12 @@ abstract class LuckyCli::Task
     end
   end
 
+  macro name(name_text)
```

Comment:
Actually I just realized this could use some docs. Now that the api is stabilizing, I think it’d be good to have docs in place

Reply:
Sweet. I wanted to look into docs for a bit anyway. I’ll get them in tomorrow.